docs/Spark-Parquet-Data-Loading.md

```diff
@@ -0,0 +1,94 @@
+# Study View Customization with Spark & Parquet
+
+A Spark implementation of data retrieval is provided for Study View for larger datasets.
+Spark loads data from Parquet files that use a columnar storage format.
+
+## Writing Parquet Files
+
+A utility is provided to write TSV files into Parquet files.
+The naming convention for Parquet files are detailed [here](#organization-of-parquet-files).
+ 
+### Running ParquetWriter with Arguments
+**--input-file** `<path to the TSV file>`<br>
+**--output-file** `<path to write Parquet file>`<br>
+**--type** `<type of file (case, cna, meta, panel, data by default)>`<br>
```

Comment:
Lets change "type" to "input-file-type".  As a description, it may be helpful to reference the Parquet file names table and (add a "Parquet Type" column to the table):
"<type of Parquet file> - see previous Parquet file name table."